automation/taskcluster/decision_task.py

```diff
@@ -66,21 +67,29 @@ def pr_or_push(is_push=False):
     for variant in variants:
         assemble_task_id = taskcluster.slugId()
         build_tasks[assemble_task_id] = BUILDER.craft_assemble_task(variant)
+        build_tasks[taskcluster.slugId()] = BUILDER.craft_test_task(variant)
         architecture, build_type, _ = get_architecture_and_build_type_and_product_from_variant(
             variant
         )
-
         if (
-            is_push and
+            # is_push and
             architecture in ('aarch64', 'arm') and
-            build_type == 'releaseRaptor' and
-            SHORT_HEAD_BRANCH == 'master'
+            build_type == 'releaseRaptor' #and
+            # SHORT_HEAD_BRANCH == 'master'
```

Comment:
for these commented out features, why not just delete them?  Do we intend to solve this later, if so, please add a comment.

Reply:
Oh that was just temporary, so I can test these on PRs, instead of just on pushes to master. I reverted it